### ISSUE TITLE
Notebooks failing as result of no 'auxdata' attribute

### ISSUE THREAD

**Issue by matthewfeickert** [lukasheinrich/pyhf#56]
# Issue

While testing the Jupyter notebooks in Binder it was noticed that they fail in the `prep_data()` function as a result of the line
```python
data = source['bindata']['data'] + pdf.auxdata
```
throwing an error as `pdf` (defined at `pdf  = hfpdf(spec)`)  does not have an 'auxdata' attribute.

# Stack Trace
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-133b58fcb35a> in <module>()
     10 }
     11 
---> 12 d,pdf = prep_data(source)
     13 init_pars = pdf.config.suggested_init()
     14 par_bounds = pdf.config.suggested_bounds()

<ipython-input-2-b59dcf407366> in prep_data(source)
     30     }
     31     pdf  = hfpdf(spec)
---> 32     data = source['bindata']['data'] + pdf.auxdata
     33     return data, pdf

AttributeError: 'hfpdf' object has no attribute 'auxdata'
```

# Fix
This can be fixed by by the following line replacement
```python
data = source['bindata']['data'] + pdf.config.auxdata
```

**Comment by matthewfeickert** [comment]
As a note for myself, as I'm fixing this, the notebooks that are affected are the following:
- `histosys`
-

**Comment by matthewfeickert** [comment]
Closed by PR #59
